tests/unit/test_apply_templates.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+import pytest
+from bs4 import BeautifulSoup
+from flask import render_template
+
+from tests.unit.conftest import mock_fund, mock_round_closed, mock_round_open
+
+
+@pytest.mark.parametrize("mock_round, window_closed_visible", [(mock_round_open, False), (mock_round_closed, True)])
+def test_render_landing(apply_test_client, mock_round, window_closed_visible):
+    result = render_template("apply-landing.html", fund=mock_fund, round=mock_round)
+    assert result
+    soup = BeautifulSoup(result, "html.parser")
+    assert soup.find("span", {"class": "govuk-caption-l"}, string="Crash Test Dummy Fund " + mock_round.round_title)
```

Comment:
When we come to add e2e tests into this repository I think it will be a helpful exercisse to look at what we've covered here and see if any of the assertions make sense to just sprinkle into the "happy path" e2e test which should be a lot more flexible and less fixed on specific content or templating

Happy for this to be in for now and we'll work out what feels right for the code base as we go!

Reply:
Agreed